portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintruleset.js

```diff
@@ -472,11 +472,17 @@ var LinearInterpRuleSet = (function () {
 	this.value_key = params.value_key;
 	this.value_range = params.value_range;
 	this.log_scale = params.log_scale; // boolean
-	this.inferred_value_range;
+
+	this.rangeTypes = {
+		'SIGNED': 'SIGNED',
+		'UNSIGNED': 'UNSIGNED',
+		'NON_POSITIVE': 'NON_POSITIVE'
```

Comment:
personally I think its a bit confusing - you see POSITIVE and you think oh ok POSITIVE, then you need to see NON. its not a big deal but, just a bit of a double take. maybe POSITIVE_OR_ZERO?